recipe/build.sh

```diff
@@ -4,4 +4,5 @@
 # https://hail.is/docs/0.2/getting_started_developing.html#requirements
 pushd $SRC_DIR/hail
 make install HAIL_COMPILE_NATIVES=1 -j ${CPU_COUNT}
-popd
\ No newline at end of file
+$PYTHON -m pip install build/deploy/dist/hail-0.2.134-py3-none-any.whl --no-deps --no-build-isolation
```

Comment:
```suggestion
$PYTHON -m pip install build/deploy/dist/hail-${PKG_VERSION}-py3-none-any.whl --no-deps --no-build-isolation
```
Might save a step on the next update?